command.c

```diff
@@ -393,7 +394,7 @@ void redis_parse_cmd(struct cmd *r) {
                 if (rlen == 0 || rnarg == 0) {
                     goto error;
                 }
-                rnarg--;
+                rnarg--; // Remove command from argument counter
```

Comment:
Since "command" is used both for the first word and for the whole array, maybe we can be more specific here. Can we call it "command name" or "command type" or something like that?

```suggestion
                rnarg--; // Remove command type from argument counter
```

Reply:
Will remove